packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -157,6 +157,33 @@ class ${class_name}:
             result[k - 1] = v
         return Output._make(result)
 
+    def _validate_parameters(self, parameter_values: List[Any]) -> List[Any]:
```

Comment:
This isn't doing validation -> it's doing type shifting....

Would `_ensure_parameter_types` or `_convert_paths_to_strings` be a better name?

Reply:
Sure. I have renamed to `_convert_paths_to_strings`.